scripts/dockerbot/check_prs.py

```diff
@@ -277,9 +313,19 @@ def check_status(pr):
                 logger.info('Setting task {} for pr {} to {}'
                             .format(task['stream'], pr['number'], states[task['result']]))
                 set_status(commit, states[task['result']], task['stream'])
+            task_states[task['stream']] = states[task['result']]
```

Comment:
What I'm saying is, if the task state is currently at "failed". When we come to "check_states" which is done every time the script is run afaicr, (it has to be, because the task result is not set by check_prs, but by each individual prt docker container when it finishes), it will still append the "failure" result and print it out each time. Or am I wrong?